lib/src/conflicts.rs

```diff
@@ -47,6 +47,14 @@ impl<T> Conflict<T> {
     pub fn adds(&self) -> &[T] {
         &self.adds
     }
+
+    pub fn set_remove(&mut self, i: usize, value: T) {
+        self.removes[i] = value;
```

Comment:
This can never resize the vector, can it? I was worried about the `self.removes[self.removes.size()]` case, and couldn't find a reference saying that panics, but the general sentiment of rusts and the docs suggests that it should.